generator-foodq/generators/app/index.js

```diff
@@ -400,6 +400,6 @@ module.exports = class extends Generator {
 	}
 
 	end() {
-		this.log('FoodQ ended', { type: "info", location: "message" });
+		this.appWizard.messages.show('FoodQ ended');
```

Comment:
wouldn't it be more clear  --> this.appWizard..showMessage('FoodQ ended');

Reply:
yes, but in the future you will have commands and other objects
now it looks similar to vscode convention

Reply:
ok